features/rails_features/auto_capture_sessions.feature

```diff
@@ -5,45 +5,18 @@ Background:
   And I set environment variable "APP_PATH" to "/usr/src"
   And I configure the bugsnag endpoint
 
-Scenario Outline: Auto_capture_sessions can be set to true in the initializer
```

Comment:
Should this test be left in (and perhaps adjusted a little) because it tests auto-capturing sessions (which your new tests don't because the new controller actions call `Bugsnag.start_session` manually)?